### PR TITLE
RK-15625 - fix:node port added to hybrid chart

### PR DESCRIPTION
**Pull request by gosharo**

### CODE REVIEW

File: charts/rookout-hybrid/templates/datastore-service.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ spec:
     - port: {{ .Values.datastore.service.port }}
       targetPort: 8080
       protocol: TCP
+    {{- if eq datastore.service.type "NodePort" }}
```
Comment (alexeygutkin) [inline charts/rookout-hybrid/templates/datastore-service.yaml:14]: we have to add it to controller too

**Review by alexeygutkin — CHANGES_REQUESTED** [review]

File: charts/rookout-hybrid/values.yaml [hunk 2]
```diff
@@ -199,6 +200,7 @@ datastore:
     type: ClusterIP
     # -- Service port
     port: 80
+    #nodePort:
```
Comment (lirook) [inline charts/rookout-hybrid/values.yaml:203]: What's this?

Comment (lirook) [reply charts/rookout-hybrid/values.yaml:203]: Let's add a better explanation here

**Review by alexeygutkin — DISMISSED** [review]

**Review by alexeygutkin — APPROVED** [review]

**Comment by gosharo** [comment]
[RK-15625](https://rookout.atlassian.net/browse/RK-15625)

**Pull request merged**